src/language/dafnyLanguageClient.ts

```diff
@@ -142,6 +142,16 @@ export class DafnyLanguageClient extends LanguageClient {
     this.diagnosticsListeners.push(callback);
   }
 
+  // Removed as of Dafny 3.8
```

Comment:
```suggestion
  // Backward compatibility for versions of Dafny <= 3.2.X
```

The removal of this API was added on July 29, 2021, before Dafny 3.3.0 was released 
https://github.com/dafny-lang/dafny/commit/5df5c01decd1ebd523f0f644e5b3af0789f6728d
(I found it using `git log -S dafny/verification/started`)

Can you fix the other one below as well, and also mention in `compilationStatusView:91` where it says `Legacy methods` to state explicitly what version of Dafny this is used for? That way, it will be easier to decide when we drop compatibility (i.e. during a major bump of the extension's version)